engine/api/workflow_queue.go

```diff
@@ -99,7 +100,7 @@ func (api *API) postTakeWorkflowJobHandler() service.Handler {
 		}
 
 		pbji := &sdk.WorkflowNodeJobRunData{}
-		report, err := takeJob(ctx, api.mustDB, api.Cache, p, id, workerModelName, pbji, wk, hatcheryName)
+		report, err := takeJob(ctx, api.mustDB, api.Cache, proj, id, workerModelName, pbji, wk, hatcheryName)
```

Comment:
need to take the proj ?

Reply:
Yes because it call UpdateNodeRunStatus that need it.